src/App.tsx

```diff
@@ -385,10 +388,17 @@ const InitialNavigator = createSwitchNavigator({
     backBehavior: 'initialRoute',
 });
 
-export default class App extends React.Component<{}, { store: any, persistor: Persistor | null }> {
-    public state = {
+interface ApplicationState {
+    store: any;
+    persistor: Persistor | null;
+    nativeAppState: AppStateStatus;
+}
+
+export default class App extends React.Component<{}, ApplicationState> {
```

Comment:
We could call this `FelfeleApp` and its state to just `State` or `FelfeleAppState`.